vuu-ui/packages/vuu-filters/src/filter-clause/ExpandoCombobox.tsx

```diff
@@ -102,16 +105,17 @@ export const ExpandoCombobox = forwardRef(function ExpandoCombobox<
       ...inputPropsProp,
       onFocus: (evt) => {
         inputPropsProp?.onFocus?.(evt);
+        if (!defaultDropdown && !touched) {
```

Comment:
I don't think the touched concept is working fully as intended. I understand the intention = disable dropdown on initial focus, but once user starts interacting with control, apply auto dropdown on entry as usual.
The reason it isn't working is that as soon as user backspaces and removes operator, the value expando is removed. When operator is selected, a brand new value expando is created, with brand new state, so back to untouched. 
This can easily be fixed, lift the `touched` state (using a ref, not state) up to the FilterClause level (injecting an onFocus to the inputProps) and use this to force showDefaultDropDown to true once touched